gamedata.cpp

```diff
@@ -2210,7 +2210,7 @@ ItemType id[] =
 	{"windchime","windchimes","WCHM",
 	 ItemType::DISABLED | ItemType::NOMARKET | ItemType::NOTRANSPORT,
 	 NULL,0,0,0, {{-1,0},{-1,0},{-1,0},{-1,0}},
-	 "CRWC",1,20, {{I_IRON,1},{I_SILVER,500},{-1,0},{-1,0}},
+	 "CRWC",1,30, {{I_SILVER,500},{-1,0},{-1,0},{-1,0}},
```

Comment:
should we increase % chance to 40%? I feel like even 30% is too low for such items